config/crd/bases/config.nri_balloonspolicies.yaml

```diff
@@ -265,6 +265,9 @@ spec:
                               description: EnergyPerformancePreference for CPUs in
                                 this class.
                               type: integer
+                            freqGovernor:
```

Comment:
So if I gather the context correctly, this would go under (bear me with the ad-hoc hierarchical syntax here) `control/cpu/class/$name/freqGovernor` , right ?

Reply:
I should have included an example, but you are right. Here is an example:
```
  control:
    cpu:
      classes:
        dynamic:
          freqGovernor: powersave
          minFreq: 800000
          maxFreq: 3600000
```